### ISSUE TITLE
SublimeLinter Doesn't Work ^4.0.0

### ISSUE THREAD

**Issue by aolsenjazz** [electron-react-boilerplate/electron-react-boilerplate#3085]
## Prerequisites

- [x] Using npm
- [x] Using an up-to-date [`main` branch](https://github.com/electron-react-boilerplate/electron-react-boilerplate/tree/main)
- [x] Using latest version of devtools. [Check the docs for how to update](https://electron-react-boilerplate.js.org/docs/dev-tools/)
- [x] Tried solutions mentioned in [#400](https://github.com/electron-react-boilerplate/electron-react-boilerplate/issues/400)
- [x] For issue in production release, add devtools output of `DEBUG_PROD=true npm run build && npm start`

## Expected Behavior

Sublime-eslint highlights errors and warnings. The following screenshot is using ERB v3.0.2
<img width="647" alt="Screen Shot 2021-12-28 at 11 16 52 PM" src="https://user-images.githubusercontent.com/13665641/147626985-f84683fd-8f12-474f-8b6d-68dc0e1c316d.png">

## Current Behavior

Sublime-eslint does not highlight errors and warnings. The following screen is using ERB v4.0.0 (and/or v4.5.0)
<img width="649" alt="Screen Shot 2021-12-28 at 11 12 23 PM" src="https://user-images.githubusercontent.com/13665641/147626817-d7f2209e-2d8d-4321-b594-c3a7b176c847.png">
_The highlight under `whoopsies` is from the Typescript plugin_

## Steps to Reproduce

1. Clone into, set up new project
````
git clone --depth 1 --branch main https://github.com/electron-react-boilerplate/electron-react-boilerplate.git my-project
cd my-project
npm i
````

2. Open sublime, open path/to/my-project

3. Make an error in src/renderer/App.tsx e.g:
````
import React from 'react';
whoopsies
import { MemoryRouter as Router, Switch, Route } from 'react-router-dom';
````

## Context

Linting works marvelously for ERB 3, but not for ^4.0.0. Poured through the links available in [Editor Configuration](https://electron-react-boilerplate.js.org/docs/editor-configuration) to no avail.

Running 'SublimeLinter: Lint this view' via Package Control results in the following console output:
<img width="1440" alt="Screen Shot 2021-12-28 at 11 20 41 PM" src="https://user-images.githubusercontent.com/13665641/147627118-1782d23f-1048-4d12-ab8a-f5b4133e8148.png">

## Your Environment

Tried with fresh install of Sublime build 4126, fresh install of SublimeLinter and SublimeLinter-eslint.

- Node version : 16.13.1
- electron-react-boilerplate version or branch : ^4.0.0
- Operating System and version : macOS 11.6
- Link to your project : n/a

Thank you to all of the maintainers of this project! Beyond grateful for the work y'all do.

**Comment by amilajack** [comment]
Linting works for vscode. I suspect this is an issue with SublimeLinter. Closing since this is out of scope of ERB

**Comment by aolsenjazz** [comment]
For anybody else who runs into this issue, this can be fixed by added [@typescript-eslint/parser](https://www.npmjs.com/package/@typescript-eslint/parser) and [@typescript-eslint/eslint-plugin](https://www.npmjs.com/package/@typescript-eslint/eslint-plugin) to dev dependencies.

**Comment by amilajack** [comment]
Fixed in 071a6fad80c86c04fe242b0239cd6bcdfa153cc1
